package.json

```diff
@@ -27,7 +27,6 @@
     "npm": ">=2.1.5"
   },
   "devDependencies": {
-    "pleeease": "3.2.0",
```

Comment:
think we need to keep this further up (or remove it if pleeease is now working as expected?)